src/pytesseract.py

```diff
@@ -94,55 +102,54 @@ def prepare(image):
 
     raise TypeError('Unsupported image object')
 
+def run_and_get_output(image, extension, lang=None, config=None, nice=None):
+    temp_name = ''
```

Comment:
Avoids additional exceptions in the stack trace if an exception occurs in save_image